ui/src/components/SearchControls/SearchControls.js

```diff
@@ -9,28 +9,19 @@ import { SearchControlsMobile } from './SearchControlsMobile';
 import { SearchPagination } from './components';
 
 export class SearchControls extends Component {
-  renderCount = totalResults => {
-    return (
-      <div className={'search-results-counter'}>
-        {totalResults} results found
-      </div>
-    );
-  };
-
   render() {
     return (
       <>
         <Responsive {...Responsive.onlyComputer}>
           <Grid columns={3} className={'search-controls'}>
             <Grid.Column largeScreen={5} computer={6}>
+              <div style={{ marginBottom: '10px' }}>
```

Comment:
please don't use inline styles (they will be hard to find and impossible to overwrite in theming) and try to limit the usage of absolute CSS units to when it is last resort (it causes usually problems with responsive layouts)
instead of that use one of existing CSS classes or if not available, create a new one in `ui/semantic-ui/.../site.overwrites`